2024-03-14 12:44:13,620 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-14 12:44:13,620 - INFO - [33mPress CTRL+C to quit[0m
2024-03-14 12:44:37,354 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-14 12:44:37,354 - INFO - [33mPress CTRL+C to quit[0m
2024-03-14 12:44:51,341 - INFO - 127.0.0.1 - - [14/Mar/2024 12:44:51] "POST /login HTTP/1.1" 200 -
2024-03-14 12:47:32,467 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-14 12:47:32,467 - INFO - [33mPress CTRL+C to quit[0m
2024-03-14 12:47:59,323 - INFO - 127.0.0.1 - - [14/Mar/2024 12:47:59] "GET /events HTTP/1.1" 200 -
2024-03-14 12:48:25,849 - INFO - User details accessed
2024-03-14 12:48:25,859 - INFO - 127.0.0.1 - - [14/Mar/2024 12:48:25] "GET /users/1 HTTP/1.1" 200 -
2024-03-14 12:48:37,911 - INFO - User details accessed
2024-03-14 12:48:37,914 - INFO - 127.0.0.1 - - [14/Mar/2024 12:48:37] "GET /users/2 HTTP/1.1" 200 -
2024-03-14 12:54:47,477 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-14 12:54:47,477 - INFO - [33mPress CTRL+C to quit[0m
2024-03-14 12:54:54,842 - INFO - User, johndoe@gmail.com, details accessed
2024-03-14 12:54:54,852 - INFO - 127.0.0.1 - - [14/Mar/2024 12:54:54] "GET /users/2 HTTP/1.1" 200 -
2024-03-14 12:44:13,620 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-14 12:44:13,620 - INFO - [33mPress CTRL+C to quit[0m
2024-03-14 12:44:37,354 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-14 12:44:37,354 - INFO - [33mPress CTRL+C to quit[0m
2024-03-14 12:44:51,341 - INFO - 127.0.0.1 - - [14/Mar/2024 12:44:51] "POST /login HTTP/1.1" 200 -
2024-03-14 12:47:32,467 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
2024-03-15 06:41:05,526 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-15 06:41:05,526 - INFO - [33mPress CTRL+C to quit[0m
2024-03-15 06:53:00,311 - ERROR - Exception on /users [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jws.py", line 257, in _load
    signing_input, crypto_segment = jwt.rsplit(b".", 1)
ValueError: not enough values to unpack (expected 2, got 1)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 535, in _decode_jwt_from_config
    unverified_claims = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 151, in decode_complete
    decoded = api_jws.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jws.py", line 198, in decode_complete
    payload, signing_input, header, signature = self._load(jwt)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jws.py", line 260, in _load
    raise DecodeError("Not enough segments") from err
jwt.exceptions.DecodeError: Not enough segments
2024-03-15 06:53:01,137 - INFO - 127.0.0.1 - - [15/Mar/2024 06:53:01] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 06:53:01,139 - INFO - 127.0.0.1 - - [15/Mar/2024 06:53:01] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 06:53:01,248 - INFO - 127.0.0.1 - - [15/Mar/2024 06:53:01] "[35m[1mGET /users HTTP/1.1[0m" 500 -
2024-03-15 06:53:28,198 - INFO - 127.0.0.1 - - [15/Mar/2024 06:53:28] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 06:53:28,198 - INFO - 127.0.0.1 - - [15/Mar/2024 06:53:28] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 06:53:35,516 - INFO - 127.0.0.1 - - [15/Mar/2024 06:53:35] "POST /login HTTP/1.1" 200 -
2024-03-15 06:56:29,673 - INFO - 127.0.0.1 - - [15/Mar/2024 06:56:29] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 06:56:29,674 - INFO - 127.0.0.1 - - [15/Mar/2024 06:56:29] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 06:56:32,297 - INFO - 127.0.0.1 - - [15/Mar/2024 06:56:32] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 06:56:32,298 - INFO - 127.0.0.1 - - [15/Mar/2024 06:56:32] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 06:56:32,301 - INFO - 127.0.0.1 - - [15/Mar/2024 06:56:32] "GET /communications HTTP/1.1" 200 -
2024-03-15 06:56:32,301 - INFO - 127.0.0.1 - - [15/Mar/2024 06:56:32] "GET /allevents HTTP/1.1" 200 -
2024-03-15 06:56:32,302 - INFO - 127.0.0.1 - - [15/Mar/2024 06:56:32] "GET /users HTTP/1.1" 200 -
2024-03-15 06:56:32,722 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 06:56:32,732 - INFO - 127.0.0.1 - - [15/Mar/2024 06:56:32] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 06:56:51,503 - INFO - 127.0.0.1 - - [15/Mar/2024 06:56:51] "[35m[1mPOST /collaborations HTTP/1.1[0m" 201 -
2024-03-15 06:56:51,514 - INFO - 127.0.0.1 - - [15/Mar/2024 06:56:51] "GET /allevents HTTP/1.1" 200 -
2024-03-15 06:59:01,110 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:01] "[35m[1mPOST /communications HTTP/1.1[0m" 201 -
2024-03-15 06:59:13,475 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:13] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 06:59:13,476 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:13] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 06:59:16,331 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 06:59:16,332 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:16] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 06:59:16,334 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:16] "GET /communications HTTP/1.1" 200 -
2024-03-15 06:59:16,335 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:16] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 06:59:16,340 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:16] "GET /users HTTP/1.1" 200 -
2024-03-15 06:59:16,341 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:16] "GET /allevents HTTP/1.1" 200 -
2024-03-15 06:59:16,344 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:16] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 06:59:27,468 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:27] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 06:59:27,469 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:27] "GET /budgets HTTP/1.1" 200 -
2024-03-15 06:59:27,470 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:27] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 06:59:27,471 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:27] "GET /assets HTTP/1.1" 200 -
2024-03-15 06:59:27,472 - INFO - 127.0.0.1 - - [15/Mar/2024 06:59:27] "GET /expenses HTTP/1.1" 200 -
2024-03-15 07:01:24,713 - INFO - 127.0.0.1 - - [15/Mar/2024 07:01:24] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 07:01:24,715 - INFO - 127.0.0.1 - - [15/Mar/2024 07:01:24] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 07:01:26,491 - INFO - 127.0.0.1 - - [15/Mar/2024 07:01:26] "GET /users HTTP/1.1" 200 -
2024-03-15 07:01:26,496 - INFO - 127.0.0.1 - - [15/Mar/2024 07:01:26] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 07:01:26,496 - INFO - 127.0.0.1 - - [15/Mar/2024 07:01:26] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 07:01:26,498 - INFO - 127.0.0.1 - - [15/Mar/2024 07:01:26] "GET /communications HTTP/1.1" 200 -
2024-03-15 07:01:26,499 - INFO - 127.0.0.1 - - [15/Mar/2024 07:01:26] "GET /allevents HTTP/1.1" 200 -
2024-03-15 07:01:26,500 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 07:01:26,506 - INFO - 127.0.0.1 - - [15/Mar/2024 07:01:26] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 07:01:29,297 - INFO - 127.0.0.1 - - [15/Mar/2024 07:01:29] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 07:01:29,300 - INFO - 127.0.0.1 - - [15/Mar/2024 07:01:29] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 07:01:29,301 - INFO - 127.0.0.1 - - [15/Mar/2024 07:01:29] "GET /assets HTTP/1.1" 200 -
2024-03-15 07:01:29,301 - INFO - 127.0.0.1 - - [15/Mar/2024 07:01:29] "GET /expenses HTTP/1.1" 200 -
2024-03-15 07:01:29,302 - INFO - 127.0.0.1 - - [15/Mar/2024 07:01:29] "GET /budgets HTTP/1.1" 200 -
2024-03-15 07:02:54,200 - INFO - 127.0.0.1 - - [15/Mar/2024 07:02:54] "GET /budgets HTTP/1.1" 200 -
2024-03-15 07:02:54,212 - INFO - 127.0.0.1 - - [15/Mar/2024 07:02:54] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 07:02:54,213 - INFO - 127.0.0.1 - - [15/Mar/2024 07:02:54] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 07:02:54,214 - INFO - 127.0.0.1 - - [15/Mar/2024 07:02:54] "GET /expenses HTTP/1.1" 200 -
2024-03-15 07:02:54,215 - INFO - 127.0.0.1 - - [15/Mar/2024 07:02:54] "GET /assets HTTP/1.1" 200 -
2024-03-15 07:06:41,278 - INFO - 127.0.0.1 - - [15/Mar/2024 07:06:41] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 07:06:41,279 - INFO - 127.0.0.1 - - [15/Mar/2024 07:06:41] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 07:06:41,280 - INFO - 127.0.0.1 - - [15/Mar/2024 07:06:41] "GET /users HTTP/1.1" 200 -
2024-03-15 07:16:50,991 - INFO - 127.0.0.1 - - [15/Mar/2024 07:16:50] "[33mPUT /update_task_status/1 HTTP/1.1[0m" 404 -
2024-03-15 07:16:55,542 - INFO - 127.0.0.1 - - [15/Mar/2024 07:16:55] "[33mPUT /update_task_status/1 HTTP/1.1[0m" 404 -
2024-03-15 14:49:46,739 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-15 14:49:46,747 - INFO - [33mPress CTRL+C to quit[0m
2024-03-15 15:11:43,309 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-15 15:11:43,339 - INFO - [33mPress CTRL+C to quit[0m
2024-03-15 15:13:34,504 - INFO - 127.0.0.1 - - [15/Mar/2024 15:13:34] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 15:13:34,505 - ERROR - Exception on /users [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jws.py", line 257, in _load
    signing_input, crypto_segment = jwt.rsplit(b".", 1)
ValueError: not enough values to unpack (expected 2, got 1)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 535, in _decode_jwt_from_config
    unverified_claims = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 151, in decode_complete
    decoded = api_jws.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jws.py", line 198, in decode_complete
    payload, signing_input, header, signature = self._load(jwt)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jws.py", line 260, in _load
    raise DecodeError("Not enough segments") from err
jwt.exceptions.DecodeError: Not enough segments
2024-03-15 15:13:34,505 - INFO - 127.0.0.1 - - [15/Mar/2024 15:13:34] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 15:13:34,669 - INFO - 127.0.0.1 - - [15/Mar/2024 15:13:34] "[35m[1mGET /users HTTP/1.1[0m" 500 -
2024-03-15 15:14:54,792 - INFO - 127.0.0.1 - - [15/Mar/2024 15:14:54] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 15:14:54,792 - INFO - 127.0.0.1 - - [15/Mar/2024 15:14:54] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 15:15:54,575 - INFO - 127.0.0.1 - - [15/Mar/2024 15:15:54] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 15:15:54,575 - INFO - 127.0.0.1 - - [15/Mar/2024 15:15:54] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 15:16:33,637 - INFO - 127.0.0.1 - - [15/Mar/2024 15:16:33] "POST /login HTTP/1.1" 200 -
2024-03-15 15:17:04,628 - INFO - 127.0.0.1 - - [15/Mar/2024 15:17:04] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 15:17:04,628 - INFO - 127.0.0.1 - - [15/Mar/2024 15:17:04] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 15:18:54,064 - INFO - 127.0.0.1 - - [15/Mar/2024 15:18:54] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 15:18:54,065 - INFO - 127.0.0.1 - - [15/Mar/2024 15:18:54] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 15:19:00,674 - INFO - 127.0.0.1 - - [15/Mar/2024 15:19:00] "POST /login HTTP/1.1" 200 -
2024-03-15 15:22:16,021 - INFO - 127.0.0.1 - - [15/Mar/2024 15:22:16] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 15:22:16,022 - INFO - 127.0.0.1 - - [15/Mar/2024 15:22:16] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 15:22:16,024 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:22:16,029 - INFO - 127.0.0.1 - - [15/Mar/2024 15:22:16] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:46:24,009 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-15 15:46:24,010 - INFO - 127.0.0.1 - - [15/Mar/2024 15:46:24] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 15:46:24,010 - INFO - 127.0.0.1 - - [15/Mar/2024 15:46:24] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 15:46:24,081 - INFO - 127.0.0.1 - - [15/Mar/2024 15:46:24] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-15 15:46:24,087 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-15 15:46:24,087 - INFO - 127.0.0.1 - - [15/Mar/2024 15:46:24] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-15 15:46:24,090 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-15 15:46:24,090 - INFO - 127.0.0.1 - - [15/Mar/2024 15:46:24] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-15 15:46:24,180 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-15 15:46:24,181 - INFO - 127.0.0.1 - - [15/Mar/2024 15:46:24] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-15 15:47:19,745 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-15 15:47:19,746 - INFO - 127.0.0.1 - - [15/Mar/2024 15:47:19] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 15:47:19,747 - INFO - 127.0.0.1 - - [15/Mar/2024 15:47:19] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 15:47:19,747 - INFO - 127.0.0.1 - - [15/Mar/2024 15:47:19] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-15 15:47:39,389 - INFO - 127.0.0.1 - - [15/Mar/2024 15:47:39] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 15:47:39,390 - INFO - 127.0.0.1 - - [15/Mar/2024 15:47:39] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 15:47:45,860 - INFO - 127.0.0.1 - - [15/Mar/2024 15:47:45] "POST /login HTTP/1.1" 200 -
2024-03-15 15:47:45,871 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:47:45,876 - INFO - 127.0.0.1 - - [15/Mar/2024 15:47:45] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:48:31,579 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:48:31,582 - INFO - 127.0.0.1 - - [15/Mar/2024 15:48:31] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:48:34,046 - INFO - 127.0.0.1 - - [15/Mar/2024 15:48:34] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 15:48:34,047 - INFO - 127.0.0.1 - - [15/Mar/2024 15:48:34] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 15:48:34,047 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:48:34,051 - INFO - 127.0.0.1 - - [15/Mar/2024 15:48:34] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:49:14,963 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:49:14,969 - INFO - 127.0.0.1 - - [15/Mar/2024 15:49:14] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:49:49,057 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:49:49,061 - INFO - 127.0.0.1 - - [15/Mar/2024 15:49:49] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:50:12,075 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:50:12,082 - INFO - 127.0.0.1 - - [15/Mar/2024 15:50:12] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:50:33,653 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:50:33,657 - INFO - 127.0.0.1 - - [15/Mar/2024 15:50:33] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:52:40,054 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:52:40,060 - INFO - 127.0.0.1 - - [15/Mar/2024 15:52:40] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:53:26,756 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:53:26,762 - INFO - 127.0.0.1 - - [15/Mar/2024 15:53:26] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:54:01,594 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:54:01,602 - INFO - 127.0.0.1 - - [15/Mar/2024 15:54:01] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:54:24,297 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:54:24,305 - INFO - 127.0.0.1 - - [15/Mar/2024 15:54:24] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:55:33,231 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:55:33,407 - INFO - 127.0.0.1 - - [15/Mar/2024 15:55:33] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:55:39,377 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:55:39,387 - INFO - 127.0.0.1 - - [15/Mar/2024 15:55:39] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:56:36,263 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:56:36,269 - INFO - 127.0.0.1 - - [15/Mar/2024 15:56:36] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:56:59,263 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:56:59,270 - INFO - 127.0.0.1 - - [15/Mar/2024 15:56:59] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:57:27,264 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:57:27,275 - INFO - 127.0.0.1 - - [15/Mar/2024 15:57:27] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:57:49,265 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:57:49,275 - INFO - 127.0.0.1 - - [15/Mar/2024 15:57:49] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 15:59:47,482 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 15:59:47,489 - INFO - 127.0.0.1 - - [15/Mar/2024 15:59:47] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:00:30,152 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:00:30,162 - INFO - 127.0.0.1 - - [15/Mar/2024 16:00:30] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:01:18,501 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:01:18,510 - INFO - 127.0.0.1 - - [15/Mar/2024 16:01:18] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:01:54,598 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:01:54,607 - INFO - 127.0.0.1 - - [15/Mar/2024 16:01:54] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:02:23,960 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:02:23,964 - INFO - 127.0.0.1 - - [15/Mar/2024 16:02:23] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:02:45,632 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-15 16:02:45,633 - INFO - 127.0.0.1 - - [15/Mar/2024 16:02:45] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-15 16:02:53,007 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-15 16:02:53,008 - INFO - 127.0.0.1 - - [15/Mar/2024 16:02:53] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 16:02:53,008 - INFO - 127.0.0.1 - - [15/Mar/2024 16:02:53] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 16:02:53,009 - INFO - 127.0.0.1 - - [15/Mar/2024 16:02:53] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-15 16:03:29,249 - INFO - 127.0.0.1 - - [15/Mar/2024 16:03:29] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 16:03:29,250 - INFO - 127.0.0.1 - - [15/Mar/2024 16:03:29] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 16:03:38,750 - INFO - 127.0.0.1 - - [15/Mar/2024 16:03:38] "POST /login HTTP/1.1" 200 -
2024-03-15 16:03:38,764 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:03:38,769 - INFO - 127.0.0.1 - - [15/Mar/2024 16:03:38] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:03:59,664 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:03:59,670 - INFO - 127.0.0.1 - - [15/Mar/2024 16:03:59] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:04:02,876 - INFO - 127.0.0.1 - - [15/Mar/2024 16:04:02] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 16:04:02,876 - INFO - 127.0.0.1 - - [15/Mar/2024 16:04:02] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 16:04:02,877 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:04:02,882 - INFO - 127.0.0.1 - - [15/Mar/2024 16:04:02] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:04:41,031 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:04:41,036 - INFO - 127.0.0.1 - - [15/Mar/2024 16:04:41] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:06:41,085 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:06:41,094 - INFO - 127.0.0.1 - - [15/Mar/2024 16:06:41] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:07:52,622 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:07:52,627 - INFO - 127.0.0.1 - - [15/Mar/2024 16:07:52] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:09:06,811 - INFO - 127.0.0.1 - - [15/Mar/2024 16:09:06] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 16:09:06,811 - INFO - 127.0.0.1 - - [15/Mar/2024 16:09:06] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 16:09:59,101 - INFO - 127.0.0.1 - - [15/Mar/2024 16:09:59] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 16:09:59,105 - INFO - 127.0.0.1 - - [15/Mar/2024 16:09:59] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 16:09:59,806 - INFO - 127.0.0.1 - - [15/Mar/2024 16:09:59] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 16:09:59,807 - INFO - 127.0.0.1 - - [15/Mar/2024 16:09:59] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 16:10:06,479 - INFO - 127.0.0.1 - - [15/Mar/2024 16:10:06] "POST /login HTTP/1.1" 200 -
2024-03-15 16:10:06,490 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:10:06,495 - INFO - 127.0.0.1 - - [15/Mar/2024 16:10:06] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:15:53,292 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:15:53,329 - INFO - 127.0.0.1 - - [15/Mar/2024 16:15:53] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:16:19,319 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:16:19,324 - INFO - 127.0.0.1 - - [15/Mar/2024 16:16:19] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:16:35,585 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:16:35,595 - INFO - 127.0.0.1 - - [15/Mar/2024 16:16:35] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:16:52,218 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:16:52,227 - INFO - 127.0.0.1 - - [15/Mar/2024 16:16:52] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:17:14,174 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:17:14,183 - INFO - 127.0.0.1 - - [15/Mar/2024 16:17:14] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:17:28,719 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:17:28,728 - INFO - 127.0.0.1 - - [15/Mar/2024 16:17:28] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:18:03,388 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:18:03,397 - INFO - 127.0.0.1 - - [15/Mar/2024 16:18:03] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:18:18,450 - INFO - 127.0.0.1 - - [15/Mar/2024 16:18:18] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 16:18:18,451 - INFO - 127.0.0.1 - - [15/Mar/2024 16:18:18] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 16:18:19,068 - INFO - 127.0.0.1 - - [15/Mar/2024 16:18:19] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 16:18:19,069 - INFO - 127.0.0.1 - - [15/Mar/2024 16:18:19] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 16:18:33,674 - INFO - 127.0.0.1 - - [15/Mar/2024 16:18:33] "POST /login HTTP/1.1" 200 -
2024-03-15 16:18:33,685 - INFO - User, wazza10@manu.com, details accessed
2024-03-15 16:18:33,692 - INFO - 127.0.0.1 - - [15/Mar/2024 16:18:33] "GET /users/3 HTTP/1.1" 200 -
2024-03-15 16:18:54,993 - INFO - 127.0.0.1 - - [15/Mar/2024 16:18:54] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 16:18:54,993 - INFO - 127.0.0.1 - - [15/Mar/2024 16:18:54] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 16:18:55,716 - INFO - 127.0.0.1 - - [15/Mar/2024 16:18:55] "[33mGET /combined-data HTTP/1.1[0m" 404 -
2024-03-15 16:18:55,717 - INFO - 127.0.0.1 - - [15/Mar/2024 16:18:55] "[33mGET /allusers HTTP/1.1[0m" 404 -
2024-03-15 16:19:02,342 - INFO - 127.0.0.1 - - [15/Mar/2024 16:19:02] "POST /login HTTP/1.1" 200 -
2024-03-15 16:19:02,354 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:19:02,359 - INFO - 127.0.0.1 - - [15/Mar/2024 16:19:02] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:22:21,261 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:22:21,292 - INFO - 127.0.0.1 - - [15/Mar/2024 16:22:21] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:23:19,280 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:23:19,286 - INFO - 127.0.0.1 - - [15/Mar/2024 16:23:19] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:24:45,170 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:24:45,175 - INFO - 127.0.0.1 - - [15/Mar/2024 16:24:45] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 16:50:44,812 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-15 16:50:44,812 - INFO - 127.0.0.1 - - [15/Mar/2024 16:50:44] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-15 16:50:47,387 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-15 16:50:47,388 - INFO - 127.0.0.1 - - [15/Mar/2024 16:50:47] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-15 16:51:02,544 - INFO - 127.0.0.1 - - [15/Mar/2024 16:51:02] "POST /login HTTP/1.1" 200 -
2024-03-15 16:51:02,557 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 16:51:02,563 - INFO - 127.0.0.1 - - [15/Mar/2024 16:51:02] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 17:07:14,024 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-15 17:07:14,025 - INFO - 127.0.0.1 - - [15/Mar/2024 17:07:14] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-15 17:07:40,872 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-15 17:07:40,873 - INFO - 127.0.0.1 - - [15/Mar/2024 17:07:40] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-15 17:14:20,104 - INFO - 127.0.0.1 - - [15/Mar/2024 17:14:20] "POST /login HTTP/1.1" 200 -
2024-03-15 17:14:20,278 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 17:14:20,282 - INFO - 127.0.0.1 - - [15/Mar/2024 17:14:20] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 17:14:35,650 - INFO - 127.0.0.1 - - [15/Mar/2024 17:14:35] "POST /login HTTP/1.1" 200 -
2024-03-15 17:14:35,683 - INFO - User, wazza10@manu.com, details accessed
2024-03-15 17:14:35,691 - INFO - 127.0.0.1 - - [15/Mar/2024 17:14:35] "GET /users/3 HTTP/1.1" 200 -
2024-03-15 17:14:50,685 - INFO - 127.0.0.1 - - [15/Mar/2024 17:14:50] "POST /login HTTP/1.1" 200 -
2024-03-15 17:14:50,719 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 17:14:50,726 - INFO - 127.0.0.1 - - [15/Mar/2024 17:14:50] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 17:21:55,088 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 17:21:55,093 - INFO - 127.0.0.1 - - [15/Mar/2024 17:21:55] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 17:34:06,498 - INFO - 127.0.0.1 - - [15/Mar/2024 17:34:06] "POST /login HTTP/1.1" 200 -
2024-03-15 17:36:16,240 - ERROR - Exception on /assets [POST]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1818, in _execute_context
    context = constructor(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 1457, in _init_compiled
    l_param: List[Any] = [
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 1459, in <listcomp>
    flattened_processors[key](compiled_params[key])
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/dialects/sqlite/base.py", line 1068, in process
    raise TypeError(
TypeError: SQLite DateTime type only accepts Python datetime and date objects as input.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 170, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/harris/Development/code/phase5/DevProject/Event-Planner 6/Event-Planner/server/app.py", line 727, in post
    # Check if event exists
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/orm/scoping.py", line 597, in commit
    return self._proxied.commit()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 1972, in commit
    trans.commit(_to_root=True)
  File "<string>", line 2, in commit
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/orm/state_changes.py", line 139, in _go
    ret_value = fn(self, *arg, **kw)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 1257, in commit
    self._prepare_impl()
  File "<string>", line 2, in _prepare_impl
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/orm/state_changes.py", line 139, in _go
    ret_value = fn(self, *arg, **kw)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 1232, in _prepare_impl
    self.session.flush()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 4296, in flush
    self._flush(objects)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 4431, in _flush
    with util.safe_reraise():
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/util/langhelpers.py", line 146, in __exit__
    raise exc_value.with_traceback(exc_tb)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 4392, in _flush
    flush_context.execute()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/orm/unitofwork.py", line 466, in execute
    rec.execute(self)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/orm/unitofwork.py", line 642, in execute
    util.preloaded.orm_persistence.save_obj(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/orm/persistence.py", line 93, in save_obj
    _emit_insert_statements(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/orm/persistence.py", line 1233, in _emit_insert_statements
    result = connection.execute(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1421, in execute
    return meth(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/sql/elements.py", line 514, in _execute_on_connection
    return connection._execute_clauseelement(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1643, in _execute_clauseelement
    ret = self._execute_context(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1824, in _execute_context
    self._handle_dbapi_exception(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 2356, in _handle_dbapi_exception
    raise sqlalchemy_exception.with_traceback(exc_info[2]) from e
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1818, in _execute_context
    context = constructor(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 1457, in _init_compiled
    l_param: List[Any] = [
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 1459, in <listcomp>
    flattened_processors[key](compiled_params[key])
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/sqlalchemy/dialects/sqlite/base.py", line 1068, in process
    raise TypeError(
sqlalchemy.exc.StatementError: (builtins.TypeError) SQLite DateTime type only accepts Python datetime and date objects as input.
[SQL: INSERT INTO assets (name, quantity, start_date, end_date, availability_status, event_id) VALUES (?, ?, ?, ?, ?, ?)]
[parameters: [{'quantity': 100, 'name': 'AfriFite', 'event_id': '1', 'end_date': '03/16/2024', 'start_date': '03/09/2024', 'availability_status': 'Yes'}]]
2024-03-15 17:36:16,489 - INFO - 127.0.0.1 - - [15/Mar/2024 17:36:16] "[35m[1mPOST /assets HTTP/1.1[0m" 500 -
2024-03-15 17:42:10,198 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-15 17:42:10,198 - INFO - [33mPress CTRL+C to quit[0m
2024-03-15 17:42:14,999 - INFO - 127.0.0.1 - - [15/Mar/2024 17:42:14] "[35m[1mPOST /assets HTTP/1.1[0m" 201 -
2024-03-15 22:35:03,699 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-15 22:35:03,700 - INFO - [33mPress CTRL+C to quit[0m
2024-03-15 22:36:45,873 - INFO - 127.0.0.1 - - [15/Mar/2024 22:36:45] "POST /login HTTP/1.1" 200 -
2024-03-15 22:36:46,119 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 22:36:46,158 - INFO - 127.0.0.1 - - [15/Mar/2024 22:36:46] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 22:37:25,892 - INFO - 127.0.0.1 - - [15/Mar/2024 22:37:25] "DELETE /login HTTP/1.1" 200 -
2024-03-15 22:48:57,498 - INFO - 127.0.0.1 - - [15/Mar/2024 22:48:57] "POST /login HTTP/1.1" 200 -
2024-03-15 22:48:57,612 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 22:48:57,655 - INFO - 127.0.0.1 - - [15/Mar/2024 22:48:57] "GET /users/1 HTTP/1.1" 200 -
2024-03-15 23:57:49,190 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-15 23:57:49,191 - INFO - [33mPress CTRL+C to quit[0m
2024-03-15 23:58:54,051 - INFO - 127.0.0.1 - - [15/Mar/2024 23:58:54] "DELETE /login HTTP/1.1" 200 -
2024-03-15 23:59:02,573 - INFO - 127.0.0.1 - - [15/Mar/2024 23:59:02] "POST /login HTTP/1.1" 200 -
2024-03-15 23:59:02,718 - INFO - User, johndoe@gmail.com, details accessed
2024-03-15 23:59:02,757 - INFO - 127.0.0.1 - - [15/Mar/2024 23:59:02] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 00:04:00,512 - INFO - 127.0.0.1 - - [16/Mar/2024 00:04:00] "DELETE /login HTTP/1.1" 200 -
2024-03-16 00:04:26,213 - INFO - 127.0.0.1 - - [16/Mar/2024 00:04:26] "POST /login HTTP/1.1" 200 -
2024-03-16 00:04:26,355 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 00:04:26,398 - INFO - 127.0.0.1 - - [16/Mar/2024 00:04:26] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 00:04:36,376 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 00:04:36,417 - INFO - 127.0.0.1 - - [16/Mar/2024 00:04:36] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 00:06:10,817 - ERROR - Exception on /events [POST]
Traceback (most recent call last):
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 170, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/qadhar/lastone/Event-Planner/server/app.py", line 272, in post
    start_date = datetime.strptime(start_date_str, '%m/%d/%Y').date()
  File "/usr/lib/python3.10/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/lib/python3.10/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '2024-03-17' does not match format '%m/%d/%Y'
2024-03-16 00:06:10,837 - INFO - 127.0.0.1 - - [16/Mar/2024 00:06:10] "[35m[1mPOST /events HTTP/1.1[0m" 500 -
2024-03-16 00:11:04,820 - ERROR - Exception on /events [POST]
Traceback (most recent call last):
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 170, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/qadhar/lastone/Event-Planner/server/app.py", line 272, in post
    start_date = datetime.strptime(start_date_str, '%m/%d/%Y').date()
  File "/usr/lib/python3.10/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/lib/python3.10/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '2024-03-17' does not match format '%m/%d/%Y'
2024-03-16 00:11:04,825 - INFO - 127.0.0.1 - - [16/Mar/2024 00:11:04] "[35m[1mPOST /events HTTP/1.1[0m" 500 -
2024-03-16 00:11:19,648 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 00:11:19,697 - INFO - 127.0.0.1 - - [16/Mar/2024 00:11:19] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 00:20:40,761 - ERROR - Exception on /events [POST]
Traceback (most recent call last):
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-16 00:20:40,775 - INFO - 127.0.0.1 - - [16/Mar/2024 00:20:40] "[35m[1mPOST /events HTTP/1.1[0m" 500 -
2024-03-16 00:20:44,398 - ERROR - Exception on /events [POST]
Traceback (most recent call last):
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-16 00:20:44,409 - INFO - 127.0.0.1 - - [16/Mar/2024 00:20:44] "[35m[1mPOST /events HTTP/1.1[0m" 500 -
2024-03-16 00:22:02,865 - INFO - 127.0.0.1 - - [16/Mar/2024 00:22:02] "DELETE /login HTTP/1.1" 200 -
2024-03-16 00:22:11,903 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jws.py", line 257, in _load
    signing_input, crypto_segment = jwt.rsplit(b".", 1)
ValueError: not enough values to unpack (expected 2, got 1)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 535, in _decode_jwt_from_config
    unverified_claims = jwt.decode(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jwt.py", line 151, in decode_complete
    decoded = api_jws.decode_complete(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jws.py", line 198, in decode_complete
    payload, signing_input, header, signature = self._load(jwt)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jws.py", line 260, in _load
    raise DecodeError("Not enough segments") from err
jwt.exceptions.DecodeError: Not enough segments
2024-03-16 00:22:11,913 - INFO - 127.0.0.1 - - [16/Mar/2024 00:22:11] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-16 00:22:28,829 - INFO - 127.0.0.1 - - [16/Mar/2024 00:22:28] "POST /login HTTP/1.1" 200 -
2024-03-16 00:22:28,995 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 00:22:29,036 - INFO - 127.0.0.1 - - [16/Mar/2024 00:22:29] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 00:23:37,919 - ERROR - Exception on /events [POST]
Traceback (most recent call last):
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 170, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/qadhar/lastone/Event-Planner/server/app.py", line 272, in post
    start_date_str = data.get('start_date')
  File "/usr/lib/python3.10/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/lib/python3.10/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '2024-03-17' does not match format '%m/%d/%Y'
2024-03-16 00:23:37,925 - INFO - 127.0.0.1 - - [16/Mar/2024 00:23:37] "[35m[1mPOST /events HTTP/1.1[0m" 500 -
2024-03-16 00:23:40,231 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 00:23:40,272 - INFO - 127.0.0.1 - - [16/Mar/2024 00:23:40] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 00:25:03,598 - ERROR - Exception on /events [POST]
Traceback (most recent call last):
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 170, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/qadhar/lastone/Event-Planner/server/app.py", line 272, in post
    start_date_str = data.get('start_date')
  File "/usr/lib/python3.10/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/lib/python3.10/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '2024-02-28' does not match format '%m/%d/%Y'
2024-03-16 00:25:03,603 - INFO - 127.0.0.1 - - [16/Mar/2024 00:25:03] "[35m[1mPOST /events HTTP/1.1[0m" 500 -
2024-03-16 00:25:06,646 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 00:25:06,692 - INFO - 127.0.0.1 - - [16/Mar/2024 00:25:06] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 00:30:56,315 - INFO - 127.0.0.1 - - [16/Mar/2024 00:30:56] "[35m[1mPOST /events HTTP/1.1[0m" 201 -
2024-03-16 00:31:11,181 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 00:31:11,216 - INFO - 127.0.0.1 - - [16/Mar/2024 00:31:11] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 00:31:59,360 - INFO - 127.0.0.1 - - [16/Mar/2024 00:31:59] "DELETE /login HTTP/1.1" 200 -
2024-03-16 00:32:06,487 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jws.py", line 257, in _load
    signing_input, crypto_segment = jwt.rsplit(b".", 1)
ValueError: not enough values to unpack (expected 2, got 1)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 535, in _decode_jwt_from_config
    unverified_claims = jwt.decode(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jwt.py", line 151, in decode_complete
    decoded = api_jws.decode_complete(
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jws.py", line 198, in decode_complete
    payload, signing_input, header, signature = self._load(jwt)
  File "/home/qadhar/.local/share/virtualenvs/server-gZuDoXi0/lib/python3.10/site-packages/jwt/api_jws.py", line 260, in _load
    raise DecodeError("Not enough segments") from err
jwt.exceptions.DecodeError: Not enough segments
2024-03-16 00:32:06,489 - INFO - 127.0.0.1 - - [16/Mar/2024 00:32:06] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-16 00:32:16,995 - INFO - 127.0.0.1 - - [16/Mar/2024 00:32:16] "POST /login HTTP/1.1" 200 -
2024-03-16 00:32:17,140 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 00:32:17,174 - INFO - 127.0.0.1 - - [16/Mar/2024 00:32:17] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 00:33:14,105 - INFO - 127.0.0.1 - - [16/Mar/2024 00:33:14] "[35m[1mPOST /events HTTP/1.1[0m" 201 -
2024-03-16 00:33:16,509 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 00:33:16,559 - INFO - 127.0.0.1 - - [16/Mar/2024 00:33:16] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 00:35:11,414 - INFO - 127.0.0.1 - - [16/Mar/2024 00:35:11] "GET /users HTTP/1.1" 200 -
2024-03-16 00:35:11,418 - INFO - 127.0.0.1 - - [16/Mar/2024 00:35:11] "GET /communications HTTP/1.1" 200 -
2024-03-16 00:35:11,424 - INFO - 127.0.0.1 - - [16/Mar/2024 00:35:11] "GET /allevents HTTP/1.1" 200 -
2024-03-16 00:35:11,446 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 00:35:11,483 - INFO - 127.0.0.1 - - [16/Mar/2024 00:35:11] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 00:35:47,093 - INFO - 127.0.0.1 - - [16/Mar/2024 00:35:47] "[35m[1mPOST /communications HTTP/1.1[0m" 201 -
2024-03-16 00:35:54,336 - INFO - 127.0.0.1 - - [16/Mar/2024 00:35:54] "GET /users HTTP/1.1" 200 -
2024-03-16 00:35:54,437 - INFO - 127.0.0.1 - - [16/Mar/2024 00:35:54] "GET /communications HTTP/1.1" 200 -
2024-03-16 00:35:54,499 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 00:35:54,518 - INFO - 127.0.0.1 - - [16/Mar/2024 00:35:54] "GET /allevents HTTP/1.1" 200 -
2024-03-16 00:35:54,558 - INFO - 127.0.0.1 - - [16/Mar/2024 00:35:54] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 09:06:42,322 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-16 09:06:42,344 - INFO - [33mPress CTRL+C to quit[0m
2024-03-16 09:09:09,942 - INFO - 127.0.0.1 - - [16/Mar/2024 09:09:09] "POST /login HTTP/1.1" 200 -
2024-03-16 09:09:09,968 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 09:09:09,973 - INFO - 127.0.0.1 - - [16/Mar/2024 09:09:09] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 09:09:14,568 - INFO - 127.0.0.1 - - [16/Mar/2024 09:09:14] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 09:09:17,222 - INFO - 127.0.0.1 - - [16/Mar/2024 09:09:17] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 09:13:04,907 - INFO - 127.0.0.1 - - [16/Mar/2024 09:13:04] "[35m[1mPOST /events HTTP/1.1[0m" 201 -
2024-03-16 09:20:31,395 - INFO - 127.0.0.1 - - [16/Mar/2024 09:20:31] "[35m[1mPOST /events HTTP/1.1[0m" 201 -
2024-03-16 09:26:26,050 - ERROR - Exception on /events/1 [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-16 09:26:26,226 - INFO - 127.0.0.1 - - [16/Mar/2024 09:26:26] "[35m[1mGET /events/1 HTTP/1.1[0m" 500 -
2024-03-16 09:27:38,373 - ERROR - Exception on /users/1 [GET]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-16 09:27:38,373 - INFO - 127.0.0.1 - - [16/Mar/2024 09:27:38] "[35m[1mGET /users/1 HTTP/1.1[0m" 500 -
2024-03-16 09:33:46,038 - INFO - 127.0.0.1 - - [16/Mar/2024 09:33:46] "POST /login HTTP/1.1" 200 -
2024-03-16 09:33:46,054 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 09:33:46,063 - INFO - 127.0.0.1 - - [16/Mar/2024 09:33:46] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 09:33:50,590 - INFO - 127.0.0.1 - - [16/Mar/2024 09:33:50] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 09:33:50,636 - INFO - 127.0.0.1 - - [16/Mar/2024 09:33:50] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 09:36:44,414 - INFO - 127.0.0.1 - - [16/Mar/2024 09:36:44] "POST /login HTTP/1.1" 200 -
2024-03-16 09:36:44,441 - INFO - User, janedee@gmail.com, details accessed
2024-03-16 09:36:44,445 - INFO - 127.0.0.1 - - [16/Mar/2024 09:36:44] "GET /users/2 HTTP/1.1" 200 -
2024-03-16 09:37:19,946 - INFO - 127.0.0.1 - - [16/Mar/2024 09:37:19] "GET /events/4 HTTP/1.1" 200 -
2024-03-16 09:37:32,697 - INFO - 127.0.0.1 - - [16/Mar/2024 09:37:32] "POST /login HTTP/1.1" 200 -
2024-03-16 09:37:32,712 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 09:37:32,720 - INFO - 127.0.0.1 - - [16/Mar/2024 09:37:32] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 09:37:36,621 - INFO - 127.0.0.1 - - [16/Mar/2024 09:37:36] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 09:37:36,625 - INFO - 127.0.0.1 - - [16/Mar/2024 09:37:36] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 09:37:50,238 - INFO - 127.0.0.1 - - [16/Mar/2024 09:37:50] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 09:37:50,241 - INFO - 127.0.0.1 - - [16/Mar/2024 09:37:50] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 09:42:51,112 - INFO - 127.0.0.1 - - [16/Mar/2024 09:42:51] "POST /login HTTP/1.1" 200 -
2024-03-16 09:42:51,134 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 09:42:51,141 - INFO - 127.0.0.1 - - [16/Mar/2024 09:42:51] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 09:43:36,974 - INFO - 127.0.0.1 - - [16/Mar/2024 09:43:36] "POST /login HTTP/1.1" 200 -
2024-03-16 09:43:36,987 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 09:43:36,995 - INFO - 127.0.0.1 - - [16/Mar/2024 09:43:36] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 09:43:41,782 - INFO - 127.0.0.1 - - [16/Mar/2024 09:43:41] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 09:43:41,786 - INFO - 127.0.0.1 - - [16/Mar/2024 09:43:41] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 09:43:48,858 - INFO - 127.0.0.1 - - [16/Mar/2024 09:43:48] "[35m[1mPOST /assignments HTTP/1.1[0m" 201 -
2024-03-16 09:43:48,874 - INFO - 127.0.0.1 - - [16/Mar/2024 09:43:48] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 09:43:48,908 - INFO - 127.0.0.1 - - [16/Mar/2024 09:43:48] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 09:43:51,275 - INFO - 127.0.0.1 - - [16/Mar/2024 09:43:51] "[35m[1mPOST /assignments HTTP/1.1[0m" 201 -
2024-03-16 09:43:51,284 - INFO - 127.0.0.1 - - [16/Mar/2024 09:43:51] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 09:43:51,313 - INFO - 127.0.0.1 - - [16/Mar/2024 09:43:51] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 09:44:02,128 - INFO - 127.0.0.1 - - [16/Mar/2024 09:44:02] "POST /login HTTP/1.1" 200 -
2024-03-16 09:44:02,143 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 09:44:02,151 - INFO - 127.0.0.1 - - [16/Mar/2024 09:44:02] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 09:48:45,419 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 09:48:45,493 - INFO - 127.0.0.1 - - [16/Mar/2024 09:48:45] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 09:50:55,668 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 09:50:55,676 - INFO - 127.0.0.1 - - [16/Mar/2024 09:50:55] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 09:53:09,393 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 09:53:09,397 - INFO - 127.0.0.1 - - [16/Mar/2024 09:53:09] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 10:00:38,648 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-16 10:00:38,648 - INFO - [33mPress CTRL+C to quit[0m
2024-03-16 10:01:27,520 - ERROR - Exception on /tasks/2 [PATCH]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-16 10:01:27,601 - INFO - 127.0.0.1 - - [16/Mar/2024 10:01:27] "[35m[1mPATCH /tasks/2 HTTP/1.1[0m" 500 -
2024-03-16 10:04:23,343 - ERROR - Exception on /tasks/2 [PATCH]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 167, in decorator
    verify_jwt_in_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 94, in verify_jwt_in_request
    jwt_data, jwt_header, jwt_location = _decode_jwt_from_request(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 340, in _decode_jwt_from_request
    decoded_token = decode_token(encoded_token, csrf_token)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/utils.py", line 128, in decode_token
    return jwt_manager._decode_jwt_from_config(encoded_token, csrf_value, allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/jwt_manager.py", line 556, in _decode_jwt_from_config
    return _decode_jwt(**kwargs, allow_expired=allow_expired)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/tokens.py", line 95, in _decode_jwt
    decoded_token = jwt.decode(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 210, in decode
    decoded = self.decode_complete(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 162, in decode_complete
    self._validate_claims(
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 248, in _validate_claims
    self._validate_exp(payload, now, leeway)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/jwt/api_jwt.py", line 306, in _validate_exp
    raise ExpiredSignatureError("Signature has expired")
jwt.exceptions.ExpiredSignatureError: Signature has expired
2024-03-16 10:04:23,344 - INFO - 127.0.0.1 - - [16/Mar/2024 10:04:23] "[35m[1mPATCH /tasks/2 HTTP/1.1[0m" 500 -
2024-03-16 10:04:55,089 - INFO - 127.0.0.1 - - [16/Mar/2024 10:04:55] "POST /login HTTP/1.1" 200 -
2024-03-16 10:04:55,111 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 10:04:55,117 - INFO - 127.0.0.1 - - [16/Mar/2024 10:04:55] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 10:05:14,178 - ERROR - Exception on /tasks/2 [PATCH]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 170, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/harris/Development/code/phase5/DevProject/Event-Planner 6/Event-Planner/server/app.py", line 481, in patch
    task.start_date = datetime.strptime(data.get('start_date', task.start_date), '%m/%d/%Y').date()
TypeError: strptime() argument 1 must be str, not datetime.date
2024-03-16 10:05:14,179 - INFO - 127.0.0.1 - - [16/Mar/2024 10:05:14] "[35m[1mPATCH /tasks/2 HTTP/1.1[0m" 500 -
2024-03-16 10:10:51,314 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-16 10:10:51,314 - INFO - [33mPress CTRL+C to quit[0m
2024-03-16 10:11:07,428 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 10:11:07,444 - INFO - 127.0.0.1 - - [16/Mar/2024 10:11:07] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 10:11:16,676 - INFO - 127.0.0.1 - - [16/Mar/2024 10:11:16] "POST /login HTTP/1.1" 200 -
2024-03-16 10:11:16,690 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 10:11:16,698 - INFO - 127.0.0.1 - - [16/Mar/2024 10:11:16] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 10:11:22,342 - ERROR - Exception on /tasks/2 [PATCH]
Traceback (most recent call last):
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 489, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask/views.py", line 110, in view
    return current_app.ensure_sync(self.dispatch_request)(**kwargs)  # type: ignore[no-any-return]
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_restful/__init__.py", line 604, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/harris/.local/share/virtualenvs/server-EFXEsOrV/lib/python3.10/site-packages/flask_jwt_extended/view_decorators.py", line 170, in decorator
    return current_app.ensure_sync(fn)(*args, **kwargs)
  File "/home/harris/Development/code/phase5/DevProject/Event-Planner 6/Event-Planner/server/app.py", line 481, in patch
    task.start_date = datetime.strptime(str(data.get('start_date', task.start_date)), '%m/%d/%Y').date()
  File "/usr/lib/python3.10/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/lib/python3.10/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '2023-03-05' does not match format '%m/%d/%Y'
2024-03-16 10:11:22,354 - INFO - 127.0.0.1 - - [16/Mar/2024 10:11:22] "[35m[1mPATCH /tasks/2 HTTP/1.1[0m" 500 -
2024-03-16 10:14:57,600 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-16 10:14:57,600 - INFO - [33mPress CTRL+C to quit[0m
2024-03-16 10:15:04,892 - INFO - 127.0.0.1 - - [16/Mar/2024 10:15:04] "PATCH /tasks/2 HTTP/1.1" 200 -
2024-03-16 10:15:08,818 - INFO - 127.0.0.1 - - [16/Mar/2024 10:15:08] "PATCH /tasks/1 HTTP/1.1" 200 -
2024-03-16 10:15:12,443 - INFO - 127.0.0.1 - - [16/Mar/2024 10:15:12] "PATCH /tasks/3 HTTP/1.1" 200 -
2024-03-16 10:18:05,609 - INFO - 127.0.0.1 - - [16/Mar/2024 10:18:05] "POST /login HTTP/1.1" 200 -
2024-03-16 10:18:05,624 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 10:18:05,632 - INFO - 127.0.0.1 - - [16/Mar/2024 10:18:05] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 10:18:09,315 - INFO - 127.0.0.1 - - [16/Mar/2024 10:18:09] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:18:09,317 - INFO - 127.0.0.1 - - [16/Mar/2024 10:18:09] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:18:32,194 - INFO - 127.0.0.1 - - [16/Mar/2024 10:18:32] "[35m[1mPOST /tasks HTTP/1.1[0m" 201 -
2024-03-16 10:18:32,211 - INFO - 127.0.0.1 - - [16/Mar/2024 10:18:32] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:19:09,414 - INFO - 127.0.0.1 - - [16/Mar/2024 10:19:09] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:19:09,417 - INFO - 127.0.0.1 - - [16/Mar/2024 10:19:09] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:19:58,725 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-16 10:19:58,725 - INFO - [33mPress CTRL+C to quit[0m
2024-03-16 10:20:02,781 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 10:20:02,798 - INFO - 127.0.0.1 - - [16/Mar/2024 10:20:02] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 10:20:05,951 - INFO - 127.0.0.1 - - [16/Mar/2024 10:20:05] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:20:05,952 - INFO - 127.0.0.1 - - [16/Mar/2024 10:20:05] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:20:35,872 - INFO - 127.0.0.1 - - [16/Mar/2024 10:20:35] "[35m[1mPOST /tasks HTTP/1.1[0m" 201 -
2024-03-16 10:20:35,884 - INFO - 127.0.0.1 - - [16/Mar/2024 10:20:35] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:23:37,801 - INFO - 127.0.0.1 - - [16/Mar/2024 10:23:37] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:23:37,803 - INFO - 127.0.0.1 - - [16/Mar/2024 10:23:37] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:30:49,307 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-16 10:30:49,307 - INFO - [33mPress CTRL+C to quit[0m
2024-03-16 10:31:07,503 - INFO - 127.0.0.1 - - [16/Mar/2024 10:31:07] "[31m[1mPOST /login HTTP/1.1[0m" 401 -
2024-03-16 10:31:16,694 - INFO - 127.0.0.1 - - [16/Mar/2024 10:31:16] "POST /login HTTP/1.1" 200 -
2024-03-16 10:31:16,712 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 10:31:16,721 - INFO - 127.0.0.1 - - [16/Mar/2024 10:31:16] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 10:31:43,930 - INFO - 127.0.0.1 - - [16/Mar/2024 10:31:43] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:31:43,937 - INFO - 127.0.0.1 - - [16/Mar/2024 10:31:43] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:37:26,332 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-16 10:37:26,332 - INFO - [33mPress CTRL+C to quit[0m
2024-03-16 10:39:01,852 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 10:39:01,868 - INFO - 127.0.0.1 - - [16/Mar/2024 10:39:01] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 10:39:18,555 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 10:39:18,563 - INFO - 127.0.0.1 - - [16/Mar/2024 10:39:18] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 10:39:20,397 - INFO - 127.0.0.1 - - [16/Mar/2024 10:39:20] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:39:20,398 - INFO - 127.0.0.1 - - [16/Mar/2024 10:39:20] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:39:49,113 - INFO - 127.0.0.1 - - [16/Mar/2024 10:39:49] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:39:49,115 - INFO - 127.0.0.1 - - [16/Mar/2024 10:39:49] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:40:11,179 - INFO - 127.0.0.1 - - [16/Mar/2024 10:40:11] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:40:11,180 - INFO - 127.0.0.1 - - [16/Mar/2024 10:40:11] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:41:15,959 - INFO - 127.0.0.1 - - [16/Mar/2024 10:41:15] "POST /login HTTP/1.1" 200 -
2024-03-16 10:41:15,975 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 10:41:15,985 - INFO - 127.0.0.1 - - [16/Mar/2024 10:41:15] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 10:41:20,987 - INFO - 127.0.0.1 - - [16/Mar/2024 10:41:20] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:41:20,991 - INFO - 127.0.0.1 - - [16/Mar/2024 10:41:20] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:41:46,173 - INFO - 127.0.0.1 - - [16/Mar/2024 10:41:46] "POST /login HTTP/1.1" 200 -
2024-03-16 10:41:46,200 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 10:41:46,208 - INFO - 127.0.0.1 - - [16/Mar/2024 10:41:46] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 10:41:54,858 - INFO - 127.0.0.1 - - [16/Mar/2024 10:41:54] "PATCH /tasks/2 HTTP/1.1" 200 -
2024-03-16 10:41:58,064 - INFO - 127.0.0.1 - - [16/Mar/2024 10:41:58] "PATCH /tasks/1 HTTP/1.1" 200 -
2024-03-16 10:42:00,401 - INFO - 127.0.0.1 - - [16/Mar/2024 10:42:00] "PATCH /tasks/1 HTTP/1.1" 200 -
2024-03-16 10:42:03,427 - INFO - 127.0.0.1 - - [16/Mar/2024 10:42:03] "PATCH /tasks/3 HTTP/1.1" 200 -
2024-03-16 10:42:18,972 - INFO - 127.0.0.1 - - [16/Mar/2024 10:42:18] "POST /login HTTP/1.1" 200 -
2024-03-16 10:42:18,988 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 10:42:18,997 - INFO - 127.0.0.1 - - [16/Mar/2024 10:42:18] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 10:42:22,570 - INFO - 127.0.0.1 - - [16/Mar/2024 10:42:22] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:42:22,572 - INFO - 127.0.0.1 - - [16/Mar/2024 10:42:22] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:43:01,164 - INFO - 127.0.0.1 - - [16/Mar/2024 10:43:01] "POST /login HTTP/1.1" 200 -
2024-03-16 10:43:01,179 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 10:43:01,190 - INFO - 127.0.0.1 - - [16/Mar/2024 10:43:01] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 10:43:04,942 - INFO - 127.0.0.1 - - [16/Mar/2024 10:43:04] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:43:04,945 - INFO - 127.0.0.1 - - [16/Mar/2024 10:43:04] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:44:22,766 - INFO - 127.0.0.1 - - [16/Mar/2024 10:44:22] "POST /login HTTP/1.1" 200 -
2024-03-16 10:44:22,779 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 10:44:22,789 - INFO - 127.0.0.1 - - [16/Mar/2024 10:44:22] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 10:44:26,635 - INFO - 127.0.0.1 - - [16/Mar/2024 10:44:26] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:44:26,639 - INFO - 127.0.0.1 - - [16/Mar/2024 10:44:26] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:44:32,268 - INFO - 127.0.0.1 - - [16/Mar/2024 10:44:32] "[35m[1mPOST /assignments HTTP/1.1[0m" 201 -
2024-03-16 10:44:32,287 - INFO - 127.0.0.1 - - [16/Mar/2024 10:44:32] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:44:32,305 - INFO - 127.0.0.1 - - [16/Mar/2024 10:44:32] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:45:01,767 - INFO - 127.0.0.1 - - [16/Mar/2024 10:45:01] "POST /login HTTP/1.1" 200 -
2024-03-16 10:45:01,784 - INFO - User, goat10@barca.com, details accessed
2024-03-16 10:45:01,786 - INFO - 127.0.0.1 - - [16/Mar/2024 10:45:01] "GET /users/4 HTTP/1.1" 200 -
2024-03-16 10:45:07,642 - INFO - 127.0.0.1 - - [16/Mar/2024 10:45:07] "PATCH /tasks/4 HTTP/1.1" 200 -
2024-03-16 10:45:17,586 - INFO - 127.0.0.1 - - [16/Mar/2024 10:45:17] "POST /login HTTP/1.1" 200 -
2024-03-16 10:45:17,600 - INFO - User, goat10@barca.com, details accessed
2024-03-16 10:45:17,602 - INFO - 127.0.0.1 - - [16/Mar/2024 10:45:17] "GET /users/4 HTTP/1.1" 200 -
2024-03-16 10:45:49,675 - INFO - 127.0.0.1 - - [16/Mar/2024 10:45:49] "POST /login HTTP/1.1" 200 -
2024-03-16 10:45:49,688 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 10:45:49,698 - INFO - 127.0.0.1 - - [16/Mar/2024 10:45:49] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 10:45:53,207 - INFO - 127.0.0.1 - - [16/Mar/2024 10:45:53] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:45:53,208 - INFO - 127.0.0.1 - - [16/Mar/2024 10:45:53] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:48:50,356 - INFO - 127.0.0.1 - - [16/Mar/2024 10:48:50] "POST /login HTTP/1.1" 200 -
2024-03-16 10:48:50,371 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 10:48:50,381 - INFO - 127.0.0.1 - - [16/Mar/2024 10:48:50] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 10:49:02,123 - INFO - 127.0.0.1 - - [16/Mar/2024 10:49:02] "GET /events/3 HTTP/1.1" 200 -
2024-03-16 10:49:02,125 - INFO - 127.0.0.1 - - [16/Mar/2024 10:49:02] "GET /events/2 HTTP/1.1" 200 -
2024-03-16 10:50:35,782 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 10:50:35,792 - INFO - 127.0.0.1 - - [16/Mar/2024 10:50:35] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 10:50:42,989 - INFO - 127.0.0.1 - - [16/Mar/2024 10:50:42] "PATCH /tasks/2 HTTP/1.1" 200 -
2024-03-16 10:53:11,119 - INFO - 127.0.0.1 - - [16/Mar/2024 10:53:11] "POST /login HTTP/1.1" 200 -
2024-03-16 10:53:11,136 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 10:53:11,145 - INFO - 127.0.0.1 - - [16/Mar/2024 10:53:11] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 10:53:37,759 - INFO - 127.0.0.1 - - [16/Mar/2024 10:53:37] "PATCH /tasks/2 HTTP/1.1" 200 -
2024-03-16 10:54:09,167 - INFO - 127.0.0.1 - - [16/Mar/2024 10:54:09] "POST /login HTTP/1.1" 200 -
2024-03-16 10:54:09,183 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 10:54:09,194 - INFO - 127.0.0.1 - - [16/Mar/2024 10:54:09] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 10:57:54,627 - INFO - 127.0.0.1 - - [16/Mar/2024 10:57:54] "POST /login HTTP/1.1" 200 -
2024-03-16 10:57:54,642 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 10:57:54,653 - INFO - 127.0.0.1 - - [16/Mar/2024 10:57:54] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 10:57:57,315 - INFO - 127.0.0.1 - - [16/Mar/2024 10:57:57] "GET /events/3 HTTP/1.1" 200 -
2024-03-16 10:57:57,316 - INFO - 127.0.0.1 - - [16/Mar/2024 10:57:57] "GET /events/2 HTTP/1.1" 200 -
2024-03-16 10:58:03,743 - INFO - 127.0.0.1 - - [16/Mar/2024 10:58:03] "GET /events/2 HTTP/1.1" 200 -
2024-03-16 10:58:03,743 - INFO - 127.0.0.1 - - [16/Mar/2024 10:58:03] "GET /events/3 HTTP/1.1" 200 -
2024-03-16 10:58:18,342 - INFO - 127.0.0.1 - - [16/Mar/2024 10:58:18] "PATCH /tasks/2 HTTP/1.1" 200 -
2024-03-16 10:58:24,253 - INFO - 127.0.0.1 - - [16/Mar/2024 10:58:24] "PATCH /tasks/2 HTTP/1.1" 200 -
2024-03-16 10:58:40,382 - INFO - 127.0.0.1 - - [16/Mar/2024 10:58:40] "POST /login HTTP/1.1" 200 -
2024-03-16 10:58:40,395 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 10:58:40,406 - INFO - 127.0.0.1 - - [16/Mar/2024 10:58:40] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 10:58:43,968 - INFO - 127.0.0.1 - - [16/Mar/2024 10:58:43] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 10:58:43,970 - INFO - 127.0.0.1 - - [16/Mar/2024 10:58:43] "GET /events/1 HTTP/1.1" 200 -
2024-03-16 10:59:03,862 - INFO - 127.0.0.1 - - [16/Mar/2024 10:59:03] "POST /login HTTP/1.1" 200 -
2024-03-16 10:59:03,877 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 10:59:03,885 - INFO - 127.0.0.1 - - [16/Mar/2024 10:59:03] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 11:01:57,538 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 11:01:57,544 - INFO - 127.0.0.1 - - [16/Mar/2024 11:01:57] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 11:03:02,273 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 11:03:02,281 - INFO - 127.0.0.1 - - [16/Mar/2024 11:03:02] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 11:03:24,291 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 11:03:24,299 - INFO - 127.0.0.1 - - [16/Mar/2024 11:03:24] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 11:03:31,863 - INFO - 127.0.0.1 - - [16/Mar/2024 11:03:31] "POST /login HTTP/1.1" 200 -
2024-03-16 11:03:31,885 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 11:03:31,894 - INFO - 127.0.0.1 - - [16/Mar/2024 11:03:31] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 11:07:10,503 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 11:07:10,511 - INFO - 127.0.0.1 - - [16/Mar/2024 11:07:10] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 11:07:26,219 - INFO - 127.0.0.1 - - [16/Mar/2024 11:07:26] "GET /events/3 HTTP/1.1" 200 -
2024-03-16 11:07:26,222 - INFO - 127.0.0.1 - - [16/Mar/2024 11:07:26] "GET /events/2 HTTP/1.1" 200 -
2024-03-16 11:08:31,669 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 11:08:31,678 - INFO - 127.0.0.1 - - [16/Mar/2024 11:08:31] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 11:27:14,899 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-03-16 11:27:14,929 - INFO - [33mPress CTRL+C to quit[0m
2024-03-16 11:27:25,576 - INFO - 127.0.0.1 - - [16/Mar/2024 11:27:25] "POST /login HTTP/1.1" 200 -
2024-03-16 11:27:25,594 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 11:27:25,603 - INFO - 127.0.0.1 - - [16/Mar/2024 11:27:25] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 11:27:54,769 - INFO - 127.0.0.1 - - [16/Mar/2024 11:27:54] "PATCH /tasks/2 HTTP/1.1" 200 -
2024-03-16 11:28:05,280 - INFO - 127.0.0.1 - - [16/Mar/2024 11:28:05] "POST /login HTTP/1.1" 200 -
2024-03-16 11:28:05,295 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 11:28:05,306 - INFO - 127.0.0.1 - - [16/Mar/2024 11:28:05] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 11:32:30,835 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 11:32:30,844 - INFO - 127.0.0.1 - - [16/Mar/2024 11:32:30] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 11:33:07,216 - INFO - 127.0.0.1 - - [16/Mar/2024 11:33:07] "PATCH /tasks/1 HTTP/1.1" 200 -
2024-03-16 11:33:10,311 - INFO - 127.0.0.1 - - [16/Mar/2024 11:33:10] "GET /events/2 HTTP/1.1" 200 -
2024-03-16 11:33:10,312 - INFO - 127.0.0.1 - - [16/Mar/2024 11:33:10] "GET /events/3 HTTP/1.1" 200 -
2024-03-16 11:33:24,284 - INFO - 127.0.0.1 - - [16/Mar/2024 11:33:24] "POST /login HTTP/1.1" 200 -
2024-03-16 11:33:24,304 - INFO - User, wazza10@manu.com, details accessed
2024-03-16 11:33:24,313 - INFO - 127.0.0.1 - - [16/Mar/2024 11:33:24] "GET /users/3 HTTP/1.1" 200 -
2024-03-16 11:34:25,464 - INFO - 127.0.0.1 - - [16/Mar/2024 11:34:25] "POST /login HTTP/1.1" 200 -
2024-03-16 11:34:25,483 - INFO - User, johndoe@gmail.com, details accessed
2024-03-16 11:34:25,491 - INFO - 127.0.0.1 - - [16/Mar/2024 11:34:25] "GET /users/1 HTTP/1.1" 200 -
2024-03-16 11:34:25,636 - INFO - 127.0.0.1 - - [16/Mar/2024 11:34:25] "POST /login HTTP/1.1" 200 -
2024-03-16 11:34:30,455 - INFO - 127.0.0.1 - - [16/Mar/2024 11:34:30] "GET /events/5 HTTP/1.1" 200 -
2024-03-16 11:34:30,458 - INFO - 127.0.0.1 - - [16/Mar/2024 11:34:30] "GET /events/1 HTTP/1.1" 200 -
